app/controllers/application_controller.rb

```diff
@@ -1,8 +1,20 @@
 class ApplicationController < ActionController::Base
+  include Pundit
+
   before_action :authenticate_user!
 
   before_action :configure_permitted_parameters, if: :devise_controller?
 
```

Comment:
Not sure if this is required, but the module talks about turning the program 'secure by default'. The code given is:

after_action :verify_authorized, unless: :devise_controller?
after_action :verify_policy_scoped, only: :index, unless: :devise_controller?